src/transforms/dropmissing.jl

```diff
@@ -0,0 +1,82 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    DropMissing()
+
+The transform that drops the rows with missing values.
+"""
+struct DropMissing <: Stateless end
+
+isrevertible(::Type{DropMissing}) = false
+
+apply(::DropMissing, table) = dropmissing(table), nothing
+
+
+# The following lines of code were taken from https://github.com/JuliaData/TableOperations.jl/blob/main/src/TableOperations.jl
+# filter
+struct Filter{F, T}
+    f::F
+    x::T
+end
+
+"""
+    TableOperations.filter(f, source) => TableOperations.Filter
+    source |> TableOperations.filter(f) => TableOperations.Filter
+Create a lazy wrapper that satisfies the Tables.jl interface and keeps the rows where `f(row)` is true.
+"""
+function filter end
```

Comment:
We can't simply copy/paste code from other packages over here. Please submit a separate PR for the Filter transform with tests and then after that we can look again into the DropMissing transform.

Reply:
Got it, thanks. I added a filter transform. I will submit the PR as soon as I am done writing unit tests.
  
I have a slightly off-topic question: Are TableOperations and TableTransforms packages somehow coordinated? It looks like a part of their functionality is going to overlap (now that we are beginning to non-numerical tranformations like Filter and DropMissing).